raiden/network/pathfinding.py

```diff
@@ -281,7 +281,7 @@ def configure_pfs_or_exit(
         )
         if maybe_pfs_url is None:
             raise RaidenError(
-                "The Service Registry has no registered Pathfinding Service "
+                "No registered Pathfinding service seems to be running"
```

Comment:
Is this vague on purpose? Otherwise I'd replace `seems to be` with `is`

Reply:
good point!